### PR TITLE
Add Liege Hackerspace (Belgium)

### PR DESCRIPTION
**Pull request by iooner**

### CODE REVIEW

**Comment by dbrgn** [comment]
Hi, thanks for the addition! You can disregard the validation error, it occurred because the change detection triggered for the line where you added a comma...

However, unfortunately CORS headers are not properly configured (see https://spaceapi.io/getting-started/#common-issues), this will break web apps trying to use the endpoint (https://spaceapi.io/validator/ for example).

To fix, you should set the `Access-Control-Allow-Origin` to `*`.

File: directory.json [hunk 1]
```diff
@@ -182,5 +182,6 @@
   "UN-Hack-Bar":"https://keinanschluss.un-hack-bar.de/spaceapi.json",
   "Zeus WPI":"https://mattermore.zeus.gent/spaceapi.json",
   "z-Labor Zwickau":"http://api.service.z-labor.space/spaceapi.json",
-  "SchonungsLos":"https://www.hackerspace-sw.de/spaceapi.json"
+  "SchonungsLos":"https://www.hackerspace-sw.de/spaceapi.json",
+  "Liege Hackerspace":"https://liegehacker.space/spaceapi/space.json"
```
Comment (rnestler) [inline directory.json:186]: Hmm it looks like the entries are partially alphabetically sorted (with capital letters first as it seems). Should we try to keep it sorted?

**Comment by rnestler** [comment]
@iooner Did you close the pull request on accident?

**Comment by gidsi** [comment]
> @iooner Did you close the pull request on accident?

#140 :)

**Pull request closed without merging**